chef_master/source/chef_system_requirements.rst

```diff
@@ -32,51 +32,44 @@ The hosted Chef server has the following requirements:
 * **Browser** --- Firefox, Google Chrome, Safari, or Internet Explorer (versions 9 or better)
 * Every node that will be configured by the chef-client and every workstation that will upload data to the Chef server must be able to communicate with the hosted Chef server
 
-Chef Server, On-premises
+Chef Server, On-premises or in cloud environment
 -----------------------------------------------------
 
 .. tag system_requirements_server_hardware
 
-All machines in a Chef server deployment (including a standalone Chef Analytics machine) have the following hardware requirements.
+All machines in a Chef server deployment have the following hardware requirements. Disk space for standalone and backend servers should scale up with the number of nodes that the servers are managing. A good rule to follow is to allocate 2 MB per node. The disk values listed below should be a good default value that you will want to modify later if/when your node count grows.
 
 For all deployments:
 
 * 64-bit architecture
 
 For a standalone deployment:
 
-* 4 total cores, 2.0 GHz AMD 41xx/61xx or Intel Xeon 5000/E5 CPUs
-* 4 GB of RAM; 8 GB of RAM for Chef Analytics
+* 4 total cores (physical or virtual)
+* 8 GB of RAM or more
 * 5 GB of free disk space in ``/opt``
 * 5 GB of free disk space in ``/var``
 
-For a tiered deployment:
+For a high availability deployment:
 
-* 8 total cores 2.0 GHz AMD 41xx/61xx or Intel Xeon 5000/E5 CPUs or faster
-* 16GB RAM
-* 2 x 300GB SAS RAID1 drives
-* Hardware RAID card
-* 1 GigE NIC interface
-* 20 GB of free disk space in ``/opt``
-* 40 GB of free disk space in ``/var``
-* A back-end server; all other systems will be front-end servers.
+* Three backend servers; as many frontend servers as required
 
-For a high availability deployment:
+.. tag system_requirements_ha
+
```

Comment:
I only ask, because [this view](https://github.com/chef/chef-web-docs/blob/9939c37c9323ee1ad5fbcf554036c2f171b1ac0d/chef_master/source/chef_system_requirements.rst) looks like it rendered an unintentional blank line 
![screen shot 2017-04-13 at 3 36 32 pm](https://cloud.githubusercontent.com/assets/382062/25025345/1db3236c-205f-11e7-8647-8ff6f8d9cd9a.png)